### PR TITLE
[#27] [UI] As a user, I can submit survey's answers

### PR DESCRIPTION
**Pull request by ducbm051291**
- #27 

## What happened

- After submit answer, the user will see Answer Submitted Successfully Screen
 
## Insight

-  Create AnswerSubmittedView
- Add Lottie in Podfile for Animation
- Create LottieView to contain animation
- Update Localizable strings and Lottie successful animation
 
## Proof Of Work

https://user-images.githubusercontent.com/24598204/210308484-a1394855-5753-4d9a-8ad3-9d0803fec840.mp4

### CODE REVIEW

File: Surveys/Sources/Presentation/Modules/AnswerSubmittedView/AnswerSubmittedView.swift [hunk 1]
```diff
@@ -0,0 +1,48 @@
+//
+//  AnswerSubmittedView.swift
+//  Surveys
+//
+//  Created by David Bui on 03/01/2023.
+//  Copyright © 2023 Nimble. All rights reserved.
+//
+
+import Kingfisher
```
Comment (blyscuit) [inline Surveys/Sources/Presentation/Modules/AnswerSubmittedView/AnswerSubmittedView.swift:9]: Do we use this?

File: Surveys/Sources/Presentation/Modules/Survey/SurveyView.swift [hunk 2]
```diff
@@ -84,9 +84,7 @@ struct SurveyView: View {
                 PrimaryButton(
                     isEnabled: .constant(true),
                     isLoading: false,
-                    action: {
-                        NotificationCenter.default.post(.unauthenticated)
-                    },
+                    action: {},
```
Comment (blyscuit) [inline Surveys/Sources/Presentation/Modules/Survey/SurveyView.swift:87]: Why remove this?

Comment (ducbm051291) [reply Surveys/Sources/Presentation/Modules/Survey/SurveyView.swift:87]: That is for test expire token ^^ I forgot to remove this in the #22. I will rebase this

File: Surveys/Sources/Presentation/Modules/AnswerSubmittedView/AnswerSubmittedView.swift (hunk 1 above)
Comment (ducbm051291) [reply Surveys/Sources/Presentation/Modules/AnswerSubmittedView/AnswerSubmittedView.swift:9]: No 😁 I'll remove this

**Review by Shayokh144 — APPROVED** [review]

**Review by blyscuit — APPROVED** [review]

**Pull request merged**